cyoa/app/models/game.rb

```diff
@@ -0,0 +1,24 @@
+class Game < ActiveRecord::Base
+  belongs_to :user
+  has_many :rounds
+
+  def create_rounds
+    10.times do
+      r = Round.new
+      r.grab_quotes
+      rounds << r
+    end
+  end
+
+  def give_uncompleted_round
+    rounds.uncompleted.first
+  end
+
+  def self.high_scores
+    Game.all.sort_by(&:total_score).reverse
```

Comment:
Anything that pulls `all` tends to be expensive. There are better ways to make this query in ActiveRecord proper.